packages/now-node/package.json

```diff
@@ -19,24 +19,28 @@
     "dist"
   ],
   "dependencies": {
-    "@types/node": "*"
+    "@types/node": "*",
+    "ts-node": "8.9.1",
+    "typescript": "3.8.3"
   },
   "devDependencies": {
     "@babel/core": "7.5.0",
     "@babel/plugin-transform-modules-commonjs": "7.5.0",
+    "@tootallnate/once": "1.1.2",
     "@types/aws-lambda": "8.10.19",
     "@types/content-type": "1.1.3",
     "@types/cookie": "0.3.3",
     "@types/etag": "1.8.0",
+    "@types/fs-extra": "^5.0.5",
     "@types/test-listen": "1.1.0",
     "@zeit/ncc": "0.20.4",
     "@zeit/node-file-trace": "0.5.1",
     "content-type": "1.0.4",
     "cookie": "0.4.0",
     "etag": "1.8.1",
+    "fs-extra": "7.0.0",
```

Comment:
Let's do it without `fs-extra` 😄

Reply:
Isn't this the same suggestion as https://github.com/zeit/now/pull/4254#discussion_r423370783? 😄

Reply:
Yep, I think we should avoid adding a new dependency in this PR